cmd/fitgen/internal/profile/codegen.go

```diff
@@ -145,16 +145,37 @@ func (g *codeGenerator) genTypes(types map[string]*Type) {
 	}
 }
 
+func (g *codeGenerator) writeDeveloperFieldDefinitions() {
+	g.p()
```

Comment:
you don't really need to define this here, you could just put it in like `dev_fields.go`

Reply:
👍 good point. Guess I just got in the code generation mentality.